test/extended/templates/templateinstance_security.go

```diff
@@ -109,17 +109,14 @@ var _ = g.Describe("[templates] templateinstance security tests", func() {
 				},
 			},
 			{
-				// at the moment, an admin cannot create a privileged object
-				// via the template instance controller as the latter only has
-				// global edit permissions.
-				by:              "checking adminuser can't create a privileged object",
+				by:              "checking adminuser can create a privileged object",
```

Comment:
yes.  the case above this one is "confirm we don't let the user do something the user isn't allowed to do"
the missing case is "confirm we don't do something we aren't allowed to do, even if the user is allowed to do it"

it ensures we're intersecting the two sets of permissions properly.